test/unit/test_event_props_reshaper.py

```diff
@@ -0,0 +1,77 @@
+from tracardi.service.event_props_reshaper import EventPropsReshaper
+from tracardi.service.notation.dot_accessor import DotAccessor
+from tracardi.domain.event import Event, EventMetadata, EventSession
+from tracardi.domain.entity import Entity
+from tracardi.domain.profile import Profile
+from tracardi.domain.session import Session, SessionMetadata
+from tracardi.domain.event_metadata import EventTime
+from tracardi.domain.event_payload_validator import ReshapeSchema
+from copy import deepcopy
+
+
+def test_should_work():
```

Comment:
This is not correct name of the test.  All test should work. It should describe what the test does.

Reply:
I will change this.